app/taran_hw20/tests/test_task.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from app.task import Task
+
+
+@pytest.mark.task_suite
+@pytest.mark.parametrize("title, description, date, priority, expected_result",
+                         [("title1", "desc1", "1111-11-11", "low",
+                           ["title1", "desc1", "1111-11-11", "low", False])])
+def test_task_initialization(title, description, date, priority, expected_result):
+    task = Task(title, description, date, priority)
+    assert task.title == expected_result[0]
+    assert task.description == expected_result[1]
+    assert task.due_date == expected_result[2]
+    assert task.priority == expected_result[3]
+    assert task.completed == expected_result[4]
+
+
+@pytest.mark.task_suite
+@pytest.mark.xfail
+@pytest.mark.parametrize("description", [("desc1")])
```

Comment:
Parameterizing is useful when you have multiple set of parameters. For example, you need to test creation of Task with 10 different Dates. 
In `test_task_initialization`, `test_task_initialization_without_title`, `test_task_initialization_without_description`, `test_task_initialization_with_required_parameters` you could implement the same test without parameterizing.